src/bin/intern.ts

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env node
 
-var fs = require('fs');
-var path = require('path');
-var cli = require('../lib/cli');
-var program = require('commander');
-var vlog = cli.getLogger();
-var print = cli.print;
-var die = cli.die;
-
-var internDir;
-var internPackage;
-
-var TESTS_DIR = 'tests';
-var MIN_VERSION = '3.0.0';
-var BROWSERS = {
+import * as fs from 'fs';
+import * as path from 'path';
+import * as cli from '../lib/cli';
+import program = require('commander'); // importing like this so helpInformation can be modified below
+import * as resolve from 'resolve';
+import { spawn } from 'child_process';
+import opn = require('opn');
+const pkg = require('../../../package.json');
```

Comment:
Make sure this works for release, especially if we don't release from the project's root directory and instead release from another location like `_build/src`